tasks/publish-to-mrrc/README.md

```diff
@@ -0,0 +1,12 @@
+# collect-mrrc-params
+
+Tekton task publish the maven artifacts to MRRC(maven.repository.redhat.com) service. MRRC is used to host maven artifacts of Red Hat Middleware products.
+This task will work with [collect-mrrc-task](../collect-mrrc-params/README.md) together to do the MRRC publishment work. It accepts the `mrrc.env` file from the [collect-mrrc-task](../collect-mrrc-params/README.md) and use the variables in it as parameters for the MRRC publishing task. It also uses `cosignPubKeyConfig` as public key to do the maven zip signature and attestation check.
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
```

Comment:
Sean just went through and standardized all of our READMEs here https://github.com/konflux-ci/release-service-catalog/pull/509/files
As such, can you add the spaces here so the following lines line up vertically? Here is an example https://raw.githubusercontent.com/seanconroy2021/release-service-catalog/b30be07186ee04d50ee5eefc45eb32c66150723d/tasks/apply-mapping/README.md

Reply:
Sure. Will fix.

Reply:
Fixed.

Reply:
Um... Seems I can not use vscode to do this alignment as it's totally mal-formatted. I've changed again use some other alignment way.

Reply:
Fixed in new commit.

Reply:
thank you!